### PR TITLE
DF-1919 - remove Labels without Mapping

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: reference-app/src/main/resources/config/liquibase/changelog/data/chisco/occupation-label-avam.csv [hunk 1]
```diff
@@ -25910,15 +25879,11 @@
...
-"AVAM-96202-it-m","96202","AVAM","it","m","studente"
-"AVAM-96203-it-m","96203","AVAM","it","m","persona in cerca di un posto di tirocinio"
 "AVAM-99601-it-m","99601","AVAM","it","m","manovale per lavori leggeri"
 "AVAM-99701-it-m","99701","AVAM","it","m","manovale per lavori normali"
 "AVAM-99801-it-m","99801","AVAM","it","m","manovale per lavori pesanti"
-"AVAM-99901-it-m","99901","AVAM","it","m","manovale per lavori molto pesanti"
-"AVAM-99999-it-m","99999","AVAM","it","m","nessuna indicazione"
+"AVAM-99901-it-m","99901","AVAM","it","m","manovale per lavori molto
+	pesanti"
 "AVAM-100000-it-m","100000","AVAM","it","m","Operatore di linee di produzione AFC"
```
Comment (tonim42) [inline reference-app/src/main/resources/config/liquibase/changelog/data/chisco/occupation-label-avam.csv]: Zeilenumbruch am falschen Ort?

Comment (fabiangotzen) [reply reference-app/src/main/resources/config/liquibase/changelog/data/chisco/occupation-label-avam.csv]: thx. fixed

Comment (fabiangotzen) [reply reference-app/src/main/resources/config/liquibase/changelog/data/chisco/occupation-label-avam.csv]: thx. fixed

**Comment by fabiangotzen** [comment]
Will be resolved with the new avam codes, was decided that it not needs to be deployed in the meantime.

**Pull request closed without merging**